src/plugins/hiddenColumns/hiddenColumns.js

```diff
@@ -604,28 +483,30 @@ class HiddenColumns extends BasePlugin {
       if (settings.copyPasteEnabled === void 0) {
         settings.copyPasteEnabled = true;
       }
+
       if (Array.isArray(settings.columns)) {
+        // this.hiddenColumnsMap.clear();
         this.hideColumns(settings.columns);
       }
+
       if (!settings.copyPasteEnabled) {
         this.addHook('modifyCopyableRange', ranges => this.onModifyCopyableRange(ranges));
```

Comment:
Is it correct that we register modifyCopyableRange on every `mapInit`? Maybe we should move it next to other hooks?

Reply:
I've done that.